### PR TITLE
Hack for matrix to vector conversion.

### PR DESCRIPTION
**Pull request by bikramkawan**
Coming from issue : https://github.com/Caleydo/taggle/issues/414

### CODE REVIEW

File: src/matrix/internal/ProjectedVector.ts [hunk 1]
```diff
@@ -34,6 +35,7 @@ export default class ProjectedVector<T, D extends IValueTypeDesc, M, MD extends
       creator: m.desc.creator,
       ts: m.desc.ts
     };
+    this.parentValueType = (<any>this.m).root.valuetype.range; //Hack to access root range for matrix to vector conversion for rendering barplot.
```
Comment (sgratzl) [inline src/matrix/internal/ProjectedVector.ts:38]: what is the difference to `this.valuetype`?

Comment (bikramkawan) [reply src/matrix/internal/ProjectedVector.ts:38]: `this.valuetype` only give the range from the matrix view created from its root matrix. While `this.parentValueType` gives the range from the parent matrix

Comment (sgratzl) [reply src/matrix/internal/ProjectedVector.ts:38]: but aren't these all the same?

Comment (bikramkawan) [reply src/matrix/internal/ProjectedVector.ts:38]: Yes you are right. I will closed this issue.

**Pull request closed without merging**